en/developer_guide/entity_list_ajax_pagination.rst

```diff
@@ -0,0 +1,45 @@
+Entity list with ajax pagination
+================================
+
+To list the entities Open Orchestra uses the  `DataTables`_ plugin.
+To improve performance, the pagination uses ajax calls.
+
+For each display of the information on the page (i.e. when paging, ordering, searching),
+the plugin makes an ajax request to the api with a number of variables to allow the required processing.
+
+The different variables useful for filtering the data (pagination, search, ordering)
+are described in this `documentation`_.
+
+In order to make the pagination operational, the api should return a facade with four information:
```

Comment:
Information is invariable.
You can say "four pieces of information", or "four fields" which seems more appropriate in the context.